packages/utils/src/media/CanvasRenderTarget.ts

```diff
@@ -71,12 +77,20 @@ export class CanvasRenderTarget
      */
     get width(): number
     {
-        return this.canvas.width;
+        if (this.canvas !== null)
+        {
+            return this.canvas.width;
+        }
+
+        throw TypeError('Canvas is null');
```

Comment:
`throw new TypeError(...)` can be more consistent.

Also maybe we can add a private method to check if the `CanvasRenderTarget` is destroyed:
```ts
get width(): number
{
    // #ifdef _DEBUG
    this._checkDestroyed();
    // #endif

    return this.canvas.width;
}

// I didn't really test the asserts, hope this will work
private _checkDestroyed(): asserts this is this & { canvas: ICanvas, context: ICanvasRenderingContext2D }
{
    if (this.canvas === null)
    {
        throw new TypeError('The CanvasRenderTarget has already been destroyed');
    }
}
```

Reply:
Yeah I think this would be a good idea as I can see this having to be used in quite a few places with any more null check changes

Reply:
The method is a good idea as this is used quite a bit